tests/vrf/vrf_config_db.j2

```diff
@@ -55,12 +64,23 @@
 {%     elif k == 'PORTCHANNEL_INTERFACE' %}
     "PORTCHANNEL_INTERFACE": {
 {%        for pc in cfg_t0['PORTCHANNEL_INTERFACE'] | sort %}
-{%             if pc in ['PortChannel0001', 'PortChannel0002'] %}
+{#     each pc have 3 keys: pc, pc|ipv4 and pc|ipv6 #}
+{%             if cfg_t0['PORTCHANNEL_INTERFACE'] | length == 12  %}
```

Comment:
I don't think it is dynamic with this check. If length is not 12, it is strictly assumed as 8 portchannels. Why don't the existing test assuming 4 PCs work? the rest can be ignored and only use PC1-4 for Vrf test, right?

Reply:
The dynamic here is using of 4 or 8 PCs, not for each number of PCs.
If you use only PC1-4 and ignore the rest, then there is no point in performing VRF tests on the T0-56 topology, which has 8 PCs.

Reply:
The dynamic here is using of 4 or 8 PCs, not for each number of PCs.
If you use only PC1-4 and ignore the rest, then there is no point in performing VRF tests on the T0-56 topology, which has 8 PCs.